packages/pwa/app/commerce-api/auth.js

```diff
@@ -490,47 +453,41 @@ export default Auth
 class Storage {
```

Comment:
Maybe beyond scope of this PR, but seems like these classes should be defined in a separate file...